zetaclient/chains/evm/observer/inbound.go

```diff
@@ -770,8 +770,8 @@ func (ob *Observer) BuildInboundVoteMsgForTokenSentToTSS(
 	)
 }
 
-// ObserveTSSReceiveInBlock queries the incoming gas asset to TSS address in a single block and posts votes
-func (ob *Observer) ObserveTSSReceiveInBlock(ctx context.Context, blockNumber uint64) error {
+// ObserveTSSReceiveInBlockAndOutTx queries the incoming gas asset to TSS address in a single block and posts votes
+func (ob *Observer) ObserveTSSReceiveInBlockAndOutTx(ctx context.Context, blockNumber uint64) error {
```

Comment:
```suggestion
func (ob *Observer) ObserveTSSReceiveInBlockAndOutbound(ctx context.Context, blockNumber uint64) error {
```
We renamed all occurrence of outTx to outbound for consistency